GoogleUtilities/AppDelegateSwizzler/Public/GoogleUtilities/GULSceneDelegateSwizzler.h

```diff
@@ -17,11 +17,11 @@
 #import <Foundation/Foundation.h>
 #import <TargetConditionals.h>
 
-#if !TARGET_OS_OSX
+#if __has_include(<UIKit/UIKit.h>)
 #import <UIKit/UIKit.h>
-#endif  // !TARGET_OS_OSX
+#endif
 
-#if ((TARGET_OS_IOS || TARGET_OS_TV) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 130000))
```

Comment:
Just double checking, we can remove `__IPHONE_OS_VERSION_MAX_ALLOWED >= 130000` because the methods below are annotated `API_AVAILABLE(ios(13.0), ...` and we're building with the iOS 17 SDK (Xcode 15), right? I'm guessing this was added when we were still supporting Xcode 10 but wanted to offer Xcode 11 (iOS 13+) features.

Reply:
Yes.